resources/views/components/layouts/app.blade.php

```diff
@@ -42,6 +42,15 @@ class="block px-4 py-2 w-full rounded-lg transition hover:bg-gray-500/10 focus:b
                     </a>
                 </li>
             @endforeach
+            <li>
+                <a
+                    href="{{ route('consulting') }}"
+                    target="_blank"
+                    class="block px-4 py-2 w-full rounded-lg transition hover:bg-gray-500/10 focus:bg-gray-500/10"
+                >
+                    Consulting
+                </a>
+            </li>
```

Comment:
Added this outside the loop to not have to refactor that part just for adding the `target` attribute to the consulting route.